components/meta-packages/hacluster-remote/hacluster-remote.p5m

```diff
@@ -0,0 +1,34 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=description value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+# main components
+depend fmri=service/cluster/service-hacluster-remote   type=require
+depend fmri=application/cluster/pacemaker              type=require
+
+# dependencies
+depend fmri=application/cluster/resource-agents               type=require
+depend fmri=application/cluster/resource-agents-openindiana   type=require
+
+# additional management gui
+depend fmri=application/cluster/pcs    type=require
```

Comment:
This gui, do we really need it? Could this be an optional dependency?

Reply:
This is not only the gui but the command line version too.
It may be an optional dependency.